npm 0.22.0，引入报错

### tdesign-mobile-vue 版本

^0.22.0

### 重现链接

_No response_

### 重现步骤

从 0.17 升级到 0.22，npm 安装无问题，安装完成后启动项目时报错如下：

补充： 0.21.0 无此问题

X [ERROR] [plugin vite:dep-pre-bundle] Cannot find module 'D:/Sites/yeqiu-h5/shared/useClickAway' from 'D:\Sites\yeqiu-h5\node_modules\tdesign-mobile-vue\es\swipe-cell\index.js'
00:32:00.921     ../../../../Sites/yeqiu-h5/node_modules/tdesign-mobile-vue/es/swipe-cell/index.js:19:7:
00:32:00.926       19 │ import '@/shared/useClickAway';
00:32:00.927          ╵        ~~~~~~~~~~~~~~~~~~~~~~~
00:32:04.173 X [ERROR] [plugin vite:dep-pre-bundle] Cannot find module 'D:/Sites/yeqiu-h5/shared' from 'D:\Sites\yeqiu-h5\node_modules\tdesign-mobile-vue\es\swipe-cell\index.js'
00:32:04.174     ../../../../Sites/yeqiu-h5/node_modules/tdesign-mobile-vue/es/swipe-cell/index.js:23:7:
00:32:04.180       23 │ import '@/shared';
00:32:04.186          ╵        ~~~~~~~~~~

### 期望结果

希望无报错

### 实际结果

_No response_

### 框架版本

_No response_

### 浏览器版本

_No response_

### 系统版本

_No response_

### Node版本

_No response_

### 补充说明

_No response_

--- a/src/shared/index.ts
+++ b/src/shared/index.ts
@@ -16,6 +16,7 @@ export * from './useTouch';
 export * from './useScrollParent';
 export * from './useExpose';
 export * from './useTest';
+export * from './useClickAway';
 
 /* directives */
 export { default as Hover } from './hover';

--- a/src/swipe-cell/swipe-cell.vue
+++ b/src/swipe-cell/swipe-cell.vue
@@ -88,11 +88,10 @@ import {
 import isFunction from 'lodash/isFunction';
 import { isArray, isBoolean } from 'lodash';
 import { useSwipe } from './useSwipe';
-import { useClickAway } from '@/shared/useClickAway';
 import props from './props';
 import config from '../config';
 import { SwipeActionItem } from './type';
-import { renderContent, renderTNode, TNode, useEmitEvent } from '../shared';
+import { renderContent, renderTNode, TNode, useEmitEvent, useClickAway } from '../shared';
 import { useSureConfirm } from './useSureConfirm';
 
 const { prefix } = config;

--- a/src/swipe-cell/useSureConfirm.ts
+++ b/src/swipe-cell/useSureConfirm.ts
@@ -11,9 +11,9 @@ import {
 } from 'vue';
 import { isFunction, isString } from 'lodash';
 import { SwipeInitData } from './swipe-cell.vue';
-import { renderTNode } from '@/shared';
+import { renderTNode } from '../shared';
 import { SwipeActionItem } from './type';
-import { TNode } from '@/common';
+import { TNode } from '../common';
 
 // 二次确认
 export function useSureConfirm(instance: ComponentInternalInstance | null, initData: SwipeInitData) {

